src/test/java/net/snowflake/ingest/streaming/internal/StreamingIngestIT.java

```diff
@@ -798,6 +798,28 @@ public void testAbortOnErrorOption() throws Exception {
     Assert.fail("Row sequencer not updated before timeout");
   }
 
+  @Test
+  public void testChannelClose() throws Exception {
+    OpenChannelRequest request1 =
+        OpenChannelRequest.builder("CHANNEL")
+            .setDBName(TEST_DB)
+            .setSchemaName(TEST_SCHEMA)
+            .setTableName(TEST_TABLE)
+            .setOnErrorOption(OpenChannelRequest.OnErrorOption.CONTINUE)
+            .build();
+
+    // Open a streaming ingest channel from the given client
+    SnowflakeStreamingIngestChannel channel1 = client.openChannel(request1);
+    for (int val = 0; val < 1000; val++) {
+      Map<String, Object> row = new HashMap<>();
+      row.put("c1", Integer.toString(val));
+      verifyInsertValidationResponse(channel1.insertRow(row, Integer.toString(val)));
+    }
+
+    // Close the channel to make sure everything is committed
+    channel1.close().get();
```

Comment:
What's the test case here?  Are we just looking to see if it errors?

Reply:
Correct, channel.close internally will check whether everything has committed, and throw an exception if not